CMSgov/onemac
# OY2 20208: Submission View: Waiver Number RAI Format

Story: https://qmacbis.atlassian.net/browse/OY2-20208
Endpoint: See github-actions bot comment

### Details
Update the hint and error messaging for the Waiver RAI Submission View Form so that submitters understand the new - format.

### Changes
- updated the text for the field hint
- added a custom message for the id validation (like what is used in Temporary Extension Form)

### Test Plan
1. Login as a submitting user
2. Navigate to Submission Dashboard
3. Select "New Submission->Waiver Action->Respond to Waiver RAI"
4. Verify the gray subtext under "Waiver Number*" matches the AC
5. Verify that after typing the territory, everything is matched to our database and the red error message appears when there is no match found.
6. Verify that if a match is found, there is no error
7. Verify that you can submit a Waiver RAI when the conditions are met.
